### ISSUE TITLE
Question about FPS

### ISSUE THREAD

**Issue by Agito555** [ZQPei/deep_sort_pytorch#163]
Hello，given a video sequence, is it normal to get the fps results range from 10 to 18 fps? 
您好，输入一段视频，得出来的FPS结果不甚高，只有10～18FPS?一般来说，FPS能达到多少呢？

**Comment by MuhammadAsadJaved** [comment]
@Agito555  Ni Hao,   did you get the FPS 10 ~ 18? 

For me using yolov3 it's 2 FPS and using Tiny-yolov3 it's about  8 FPS.

**Comment by Agito555** [comment]
Yeah, i get the 10—18 fps by using yolov3 and 20—25 FPS by using yolov3—tiny



---Original---

**Comment by MuhammadAsadJaved** [comment]
@Agito555 I do not know why it's too low for me. May I know your system specs?

**Comment by Agito555** [comment]
the question may lie in the size of the input video and which type gpu you run the demo in.&nbsp;
if my memory serves me well，i input the video of 640*360，and i run the demo in Titanic XP



---Original---

**Comment by MuhammadAsadJaved** [comment]
@Agito555  
                   My input video is 1920 x 1080,   and here are my systems specs. 
![Screenshot from 2020-08-24 20-46-41](https://user-images.githubusercontent.com/28862708/91046455-0770f380-e64b-11ea-8075-ed90bc345753.png)

**Comment by Agito555** [comment]
you can try to change to resize your input video,by which fps will be a little faster.&nbsp;
i have run the demo with the input video of 1980*1080, the fps is slower than the input video of 640*360.
i don't know whether the ability of your gpu is good enough. Maybe you can run the demo in a better gpu to examine the fps result.&nbsp;
BTW，where are you from?&nbsp;

---Original---

**Comment by MuhammadAsadJaved** [comment]
Ok Thanks. I am from Pakistan and working in Beijing.

**Comment by SuperbTUM** [comment]
Does the tracking speed have something to do with the detector? I tried with YoloV5 and got a faster tracking result.

**Comment by da1396** [comment]
> Does the tracking speed have something to do with the detector? I tried with YoloV5 and got a faster tracking result.

@SuperbTUM I want to use YOLOv5 but don't know how to make the settings(directory structure, which files to need..). Could you let me know your workaround?

**Comment by Agito555** [comment]
> to do with the detector? I tried with YoloV5 and got a faster tracking result.

Detecting the object is the first step for tracking. Faster detector is of help to improve tracking speed.
